ssr-express/README.md

```diff
@@ -0,0 +1,131 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## Introduction
```

Comment:
Is this only really useful to AMP Canonical sites? (Maybe say this?)

What happens if an AMP input file is not valid AMP?

Reply:
The first point was added on the latest commit. 

Re: the second point, the amp-ssr component doesn't perform validations, so invalid AMPs are transformed. Non-AMP pages, without ⚡, are not transformed. Those are more details of the amp-ssr component, than the middleware. We should add this explanation there and link it from this doc.